README.md

```diff
@@ -2,54 +2,76 @@
 
 # Stager
 
-Stager enables rapid Flutter development and encourages good architectural practices by allowing developers quickly launch and develop isolated portions of an app.
+Stager is a Flutter development tool that allows you to run small portions of your app as individual Flutter apps. This lets you:
 
-Stager can accelerate your development workflow for widgets that:
+- Focus your development on a single widget or flow – no more clicking through multiple screens or setting external feature flags to reach the page you're working on.
+- Ensure your UI works in *all* cases, including:
+  - Light and Dark mode
+  - Small or large text sizes
+  - Different viewport sizes
+  - Different device types
+  - Loading, empty, error, and normal states
+- Show all of this to your designers to make sure your app is pixel-perfect.
```

Comment:
but since this is the entry page, you should point out some of the other reasons why this exists, modularity, testing on platforms without all the plugins, faster build time, etc etc

Reply:
+1 to faster build time.

Maybe it makes sense to sort of answer the question "why do you need Stager when you already have hot reload?" somewhere here? Like "While hot reload already allows you to iterate quickly on a specific UI, Stager adds the ability to inject fake data, change environment parameters, and only build the part of the app that you're working on.